### PR TITLE
[WebProfilerBundle] Improved page for logs

### PR DESCRIPTION
**Pull request by hason**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

_Before_
logs-before
_After_
logs-after

### CODE REVIEW

**Comment by Rvanlaak** [comment]
:+1: nice

Maybe use [the new `dump()`](http://symfony.com/doc/current/components/var_dumper/introduction.html) to output the context?

**Comment by aitboudad** [comment]
:+1:

**Comment by fabpot** [comment]
:+1:

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig [hunk 1]
```diff
@@ -87,15 +87,30 @@
...
+                <tr class="{% if log.priority >= 400 %}error{% elseif log.priority >= 300 %}warning{% endif %}{% if log.context.scream is defined %}scream{% endif %}">
+                    <td>{{ i + 1 }}</td>
+                    <td>
+                        {%- if log.context.type is defined and (constant('E_DEPRECATED') == log.context.type or constant('E_USER_DEPRECATED') == log.context.type) -%}
+                            DEPRECATION
+                        {%- else -%}
+                            {{ log.priorityName }}
+                        {%- endif -%}
+                    </td>
+                    <td>{{ log.channel is defined ? log.channel }}</td>
```
Comment (fabpot) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig]: What about removing the column when we don't have information about the channel?

**Comment by timglabisch** [comment]
looks good but needs a lot of space. i would like to see stats about rendering performance on large logs.

**Comment by fabpot** [comment]
Can you rebase this one?

**Comment by hason** [comment]
@fabpot Rebased.

**Comment by fabpot** [comment]
Thank you @hason.

**Pull request closed without merging**